<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>meatDialog</class>
 <widget class="QDialog" name="meatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>26</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>肉制品</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>120</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请在前面打钩就可以了</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>731</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="clearPushButton">
      <property name="text">
       <string>清除</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="returnPushButton">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pPushButton">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QSpinBox" name="duckSpinBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="beefCheckBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>精牛肉</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="porkCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>精猪肉</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="duckCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>鲜鸭肉</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chickCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>鲜鸡肉</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="fishCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>鱼肉</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="muttonCheckBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>鲜羊肉</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="fishSpinBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="chickenSpinBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="porkSpinBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="beefspinBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="muttonSpinBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>311</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="picture.qrc">:/new/prefix0/resource/rou.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="paiguCheckBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>排骨</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="feirouCheckBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>350</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>肥肉</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="paiguSpinBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>290</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="feirouSpinBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>350</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections/>
</ui>
